templates/artist/artist_detail.html

```diff
@@ -47,35 +47,19 @@
 {% endblock %}
 
 {% block content %}
-    <h1 class="header-font">{{ artist.name }}</h1>
-    {% if investors or artist.past_campaigns or updates or has_permission_to_submit_update %}
```

Comment:
I think I see where this comes into play now though. I think I got this confused with an "if auth" statement and removed it by accident